libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -685,10 +682,8 @@ bool AC_WPNav::set_spline_origin_and_destination(const Vector3f& origin, const V
     // mission is "active" if wpnav has been called recently and vehicle reached the previous waypoint
     bool prev_segment_exists = (_flags.reached_destination && ((AP_HAL::millis() - _wp_last_update) < 1000));
 
-    float dt = _pos_control.time_since_last_xy_update();
-    if (dt >= 0.2f) {
```

Comment:
It's not clear to me where these sanity checks have gone.

Reply:
I don't think there's any need for the sanity check anymore because we've changed from a dt that could vary with the CPU load to a dt that is based on the desired main loop rate so it will never vary.

It's essentially a change from the actual-loop-rate to the desired-loop-rate.  The two should be very close if the flight controller is running well.